doc/cephadm/services/monitoring.rst

```diff
@@ -83,6 +83,31 @@ steps below:
 
      ceph orch apply grafana
 
+.. _cephadm-monitoring-centralized-logs:
+
+Centralized Logging in Ceph
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ceph-Dashboard now provides centralized logging solution using Loki & Promtail. Centralized Log Management (CLM) is a type
+of logging solution system that consolidates all of your log data and pushes it to one central, accessible,
+and easy-to-use interface. Centralized logging is designed to make your life easier.
+Some of the advantages of using centralized logging are:
+
+#. Linear event timeline (instead of 1000s of different logs), easier to analyze issues.
+#. Real-time live log monitoring: with 1000s of logs is impractical to follow logs.
+#. No need to jump to different hosts for troubleshooting, everything in a single location.
+#. Flexible retention policies. With per-daemon logs, log rotation is usually set to a short interval (1-2 weeks) to save disk usage.
+#. Increased security & backup: logs can contain sensitive information & usage patterns. Additionally, centralized logging allows for HA, etc.
+
+Centralized Logging in Dashboard is implemented using two new services - ``loki`` & ``promtail``.
+
+Loki is like Prometheus, but for logs. Loki is a log aggregation system designed to store and query logs from all your applications and infrastructure
+and can be configured as a Datasource in grafana.
+
+Promtail is an agent which ships the contents of local logs to a private Grafana Loki instance or Grafana Cloud.
```

Comment:
This is a verbatim quote from Grafana website (but without clear quoting), and ["are quotes protected by copyright?"](https://www.lizerbramlaw.com/2015/10/27/does-copyright-protect-quotes/). So please remove these lines and point to the Grafana website if someone needs clarification about what Loki is.
```suggestion
Centralized Logging in Dashboard is implemented using two new services:
* Loki: briefly describe loki using your own words and you may add a link to the Loki page.
* Promtail: same thing.
```